src/wizards/tokenSwapDealWizard/components/tokenDetails/tokenDetails.ts

```diff
@@ -171,6 +178,8 @@ export class TokenDetails {
   }
 
   private addValidation() {
+    if (this.form.errors.length > 0) return;
```

Comment:
what is this condition for?

Reply:
would be good to comment this.

Reply:
sorry, dev remnant. Pushed a remove commit